pkg/bootstrap/cni_test.go

```diff
@@ -31,7 +31,7 @@ func TestCNISupportedVersionRendering(t *testing.T) {
 		{
 			name:              "Default Supported Versions",
 			supportedVersions: versions.GetSupportedVersions(),
-			containerNames:    []string{"install-cni-v1-1", "install-cni-v2-0", "install-cni-v2-1", "install-cni-v2-2", "install-cni-v2-3"},
+			containerNames:    []string{"install-cni-v2-3", "install-cni-v1-1", "install-cni-v2-0", "install-cni-v2-1", "install-cni-v2-2"},
```

Comment:
Does order of these names matter?

Reply:
yes, it matters now. because it finds the v2.3 one and append it first

Reply:
Thanks!